### ISSUE TITLE
Duvidas Utilização Lib

### ISSUE THREAD

**Issue by Markosbn** [jean-merelis/coffeepot-br-sintegra#1]
boa tarde, localizei a falta do modelo de nota 65 no  enum DocumentoFiscal. Posso fazer um PR para a correção. ou mando para você os erros para serem atualizados?

**Comment by jean-merelis** [comment]
Olá Marcos! Fique a vontade em fazer o PR.
O SINTEGRA foi desobrigado em vários estados. Em qual estado você atua?

**Comment by Markosbn** [comment]
Certo, farei as alterações e o PR para você, então estamos preparando agora o sintegra para o estado de MG. Que esta ainda com a obrigatoriedade do documento.

**Comment by Markosbn** [comment]
Fiz o PR para você, pelos testes que fiz, o que faltava mesmo era o modelo de nota, caso encontre mais algo futuramente volto a  comentar com você
